doc/developers/testsuite.html

```diff
@@ -479,6 +479,11 @@ <h3>Restricting tests to feature configurations</h3>
 category/test.with_zlib=off.output
 </pre>
       These tests will only be set up if the specified feature was configured.
+<p><b>Note:</b> For CMake variables switching subpackages of third party libraries
```

Comment:
I find the term "switching" confusing here. Perhaps use "For CMake variables indicating whether sub-packages of third party libraries were detected, like ..."?

Reply:
Sounds better, I changed it to that.